gneiss/_model.py

```diff
@@ -0,0 +1,119 @@
+import pickle
+from skbio import TreeNode
+import abc
+from skbio.stats.composition import ilr_inv, closure
+import numpy as np
+import pandas as pd
+
+
+class Model(metaclass=abc.ABCMeta):
+
+    def __init__(self, submodels, basis, tree, balances):
+        """
+        Abstract container for balance models.
+
+        Parameters
+        ----------
+        submodels : list of statsmodels objects
+            List of statsmodels result objects.
+        basis : pd.DataFrame
+            Orthonormal basis in the Aitchison simplex.
+            Row names correspond to the leafs of the tree
+            and the column names correspond to the internal nodes
+            in the tree. If this is not specified, then `project` cannot
```

Comment:
The `project` parameter determines if the return type of this method should be proportions or balances.  Literally, it means should I project this into the Simplex (i.e. proportions) or balance space.  

I would be interested in recommendations for improving this aspect of the documentation - this has tripped up a few people in the past.